build

```diff
@@ -67,6 +69,7 @@ function image_name() {
 }
 
 function do_build() {
+    compute_metadata
```

Comment:
I think this is the only place that needs the variables from `compute_metadata`

Reply:
`do_test`, `do_describe` `do_push` use `image_name` that reads `$TAG_PREFIX` but it is not used in `do_tag`.
I added it for consistency but I removed it.

Reply:
Ah, thanks. I missed that part.